ffs/api/api.go

```diff
@@ -48,18 +48,28 @@ type API struct {
 
 // New returns a new Api instance.
 func New(ctx context.Context, iid ffs.APIID, is InstanceStore, sch ffs.Scheduler, wm ffs.WalletManager, dc ffs.DefaultCidConfig) (*API, error) {
+	addr, err := wm.NewAddress(ctx, defaultAddressType)
+	if err != nil {
+		return nil, fmt.Errorf("creating new wallet addr: %s", err)
+	}
+
+	dc.Cold.Filecoin.Addr = addr
```

Comment:
When we create a new ffs instance, we create a new address and set it as the default in the default config